css/properties/font-size-adjust.json

```diff
@@ -66,45 +50,29 @@
             "ie": {
               "version_added": false
             },
-            "opera": [
-              {
-                "version_added": "41"
-              },
-              {
-                "version_added": "30",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "Enable experimental Web Platform features"
-                  }
-                ]
-              }
-            ],
-            "opera_android": [
-              {
-                "version_added": "41"
-              },
-              {
-                "version_added": "30",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "Enable experimental Web Platform features"
-                  }
-                ]
-              }
-            ],
+            "opera": {
+              "version_added": "30",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "Enable experimental Web Platform features"
+                }
+              ]
+            },
+            "opera_android": {
+              "version_added": false
+            },
             "safari": {
               "version_added": false
             },
             "safari_ios": {
               "version_added": false
             },
             "samsunginternet_android": {
-              "version_added": "6.0"
+              "version_added": false
             },
             "webview_android": {
-              "version_added": "54"
+              "version_added": false
```

Comment:
This should mirror Android Chrome.

Reply:
Webview Android doesn't have flags, right?  Support for this feature is behind a flag, so `false` is accurate in this context?

Reply:
Yes. Brain fumble.

Reply:
Yep, I had to change this from the original data in 5b76cf94552b1dd91d744c6c3b94c9f481be1548 as the linter picked it up:

> This browser (webview_android) does not support flags, so support cannot be behind a flag for this feature.